### ISSUE TITLE
Document configuration options

### ISSUE THREAD

**Issue by rsvoboda** [quarkus-qe/quarkus-test-framework#369]
Document configuration options

E.g. " ts.global.delete.folder.on.exit=false" is not documented

Look at automation options

**Comment by jsmrcka** [comment]
I created a PR in a temp repository: https://github.com/jsmrcka/quarkus-test-framework-wiki/pull/1. @michalvavrik

It contains proposed changes which document missing config options.

I am not aware of a way how to make PR into a wiki, so this is a workaround. After a review, I can push identical changes directly into the TF wiki.

**Comment by michalvavrik** [comment]
Sounds good @jsmrcka .

**Comment by rsvoboda** [comment]
No PRs into wiki, just direct edits, but I like to have external view for the planned changes as in https://github.com/jsmrcka/quarkus-test-framework-wiki/pull/1. +1

**Comment by michalvavrik** [comment]
I don't agree, PR was needed, direct edit is not a good way @rsvoboda . I made several comments, some of them might be good :-)

**Comment by michalvavrik** [comment]
I checked, @jsmrcka must grant you access, I don't have rights.

**Comment by michalvavrik** [comment]
I mean PR as a way to collaborate, yeah, there is no way to make PR for Wiki AFAICT

**Comment by jsmrcka** [comment]
Changes from the PR https://github.com/jsmrcka/quarkus-test-framework-wiki/pull/1 have been pushed to the wiki main.

I'm leaving this issue open, as the automation options are still TODO.
